src/trace.js

```diff
@@ -278,6 +281,7 @@ export default {
   },
 
   resetZoom: function(chart) {
+    const xAxisName = chart.config.options.scales.x ? 'x' : chart.config.options.scales.xAxes ? 'xAxes' : undefined;
```

Comment:
same as above, just take it from the options and if not specify fall back to default `x`